src/test/rbd_mirror/image_replayer/test_mock_PrepareRemoteImageRequest.cc

```diff
@@ -328,6 +328,7 @@ TEST_F(TestMockImageReplayerPrepareRemoteImageRequest, SuccessJournal) {
   ASSERT_TRUE(mock_journal_state_builder.remote_journaler != nullptr);
   ASSERT_EQ(cls::journal::CLIENT_STATE_DISCONNECTED,
             mock_journal_state_builder.remote_client_state);
+  delete mock_journal_state_builder.remote_journaler;
```

Comment:
Since there is no corresponding `new` or `create()` in the test, it's a bit confusing.  Let's add a comment, here and in the other test:
```suggestion

  // owned by StateBuilder, normally freed in StateBuilder::close()
  delete mock_journal_state_builder.remote_journaler;
```

Reply:
Fixed, thanks~